addon-test-support/@ember/test-helpers/application.ts

```diff
@@ -11,11 +13,11 @@ var __application__;
   @public
   @param {Ember.Application} application the application that will be tested
 */
-export function setApplication(application) {
+export function setApplication(application: Application): void {
   __application__ = application;
 
   if (!getResolver()) {
-    let Resolver = application.Resolver;
+    let Resolver = (application as any).Resolver;
```

Comment:
Is there no type for `Ember.Resolver`?

Reply:
there is, and I have commits in the queue that introduce it